### PR TITLE
remove redundant code and better error msg

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: allencloud <allen.sun@daocloud.io>

**- What I did**
1. remove redundant code in swarm_unlock.go;
2. better volume prune error message;

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Review by LK4D4 — APPROVED** [review]
LGTM

File: client/volume_prune.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func (cli *Client) VolumesPrune(ctx context.Context, pruneFilters filters.Args)
 	defer ensureReaderClosed(serverResp)
 
 	if err := json.NewDecoder(serverResp.body).Decode(&report); err != nil {
-		return report, fmt.Errorf("Error retrieving disk usage: %v", err)
+		return report, fmt.Errorf("Error retrieving volume prune report:: %v", err)
```
Comment (thaJeztah) [inline client/volume_prune.go]: looks like there's a double `::` on this line now

Comment (allencloud) [inline client/volume_prune.go]: Good catch. I think it is a mistake introduced from this PR. sorry for that. And I am add a pr to fix this

Comment (thaJeztah) [inline client/volume_prune.go]: no worries, I just noticed :smile:

**Pull request merged**